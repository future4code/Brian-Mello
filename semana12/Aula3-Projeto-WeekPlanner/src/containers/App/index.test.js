import React from 'react';
import {App} from '.';
import renderer from 'react-test-renderer';

describe('Teste Do App Container', () => {
    it('Snapshot', () => {
        const app = renderer.create(<App/>).toJSON();
        expect(app).toMatchSnapshot();
    });
});