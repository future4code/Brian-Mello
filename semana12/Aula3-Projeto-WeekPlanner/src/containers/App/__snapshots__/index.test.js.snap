// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste Do App Container Snapshot 1`] = `
<div
  className="sc-EHOje iNMVPu"
>
  <header
    className="sc-bdVaJa fwxdgV"
  >
    <div
      className="sc-htoDjs dGkkfX"
    >
      <h1
        className="sc-gzVnrw bFPEn"
      >
        W E E K 
      </h1>
      <h1
        className="sc-gzVnrw bFPEn"
      >
        P L A N N E R
      </h1>
    </div>
  </header>
  <main
    className="sc-bxivhb bqEhTm"
  >
    <div
      className="sc-bwzfXH kxPUwG"
    >
      <div
        className="sc-bZQynM ihpZpy"
      >
        <form
          className="sc-gZMcBi bAHgmZ"
          onSubmit={[Function]}
        >
          <div>
            <div
              className="MuiFormControl-root-1 sc-gqjmRU gLqvCz"
            >
              <label
                className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
                data-shrink={false}
              >
                Task
                <span
                  className="MuiFormLabel-asterisk-22"
                >
                   *
                </span>
              </label>
              <div
                className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-formControl-37 MuiInput-formControl-24"
                onClick={[Function]}
              >
                <input
                  aria-invalid={false}
                  className="MuiInputBase-input-46 MuiInput-input-31"
                  disabled={false}
                  name="text"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  required={true}
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <select
            className="sc-jzJRlG CsnDN"
            name="day"
            onChange={[Function]}
            value=""
          >
            <option
              selected="selected"
              value=""
            >
              Change a day
            </option>
            <option
              value="Segunda"
            >
              Monday
            </option>
            <option
              value="Terça"
            >
              Tuesday
            </option>
            <option
              value="Quarta"
            >
              Wednesday
            </option>
            <option
              value="Quinta"
            >
              Thursday
            </option>
            <option
              value="Sexta"
            >
              Friday
            </option>
            <option
              value="Sábado"
            >
              Saturday
            </option>
            <option
              value="Domingo"
            >
              Sunday
            </option>
          </select>
          <button
            className="MuiButtonBase-root-79 MuiButton-root-53 MuiButton-text-55 MuiButton-flat-58 sc-ifAKCX iGMeTL"
            disabled={false}
            onBlur={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="submit"
          >
            <span
              className="MuiButton-label-54"
            >
              Criar Tarefa
            </span>
            <span
              className="MuiTouchRipple-root-82"
            />
          </button>
        </form>
      </div>
    </div>
    <main
      className="sc-htpNat euEwMU"
    >
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Monday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Tuesday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Wednesday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Thursday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Friday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Saturday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
      <div
        className="sc-VigVT bLQdKN"
      >
        <header
          className="sc-jTzLTM TJfAr"
        >
          Sunday
        </header>
        <hr
          className="sc-cSHVUG cGrTwm"
        />
        <main
          className="sc-fjdhpX gWzwRZ"
        />
      </div>
    </main>
  </main>
</div>
`;
